src/domain/event/utils.ts

```diff
@@ -18,3 +24,94 @@ export const getEventFields = (event: Event, locale: Language): EventFields => {
     startTime: event.start_time ? new Date(event.start_time) : null,
   };
 };
+
+export const getEventLocationText = ({
+  event,
+  locale,
+}: {
+  event: Event;
+  locale: Language;
+}): string => {
+  const { location } = event;
+  const {
+    addressLocality,
+    name: locationName,
+    streetAddress,
+  } = location
+    ? getPlaceFields(location, locale)
+    : { addressLocality: '', name: '', streetAddress: '' };
+
+  return (
+    [locationName, streetAddress, addressLocality]
+      .filter((e) => e)
+      .join(', ') || '-'
+  );
+};
+
+const getDateArray = (date: Date): DateArray => {
+  const parts = formatDate(date, 'yyyy-M-d-H-m')
+    .split('-')
+    .map((e) => Number(e));
+  return [parts[0], parts[1], parts[2], parts[3], parts[4]];
```

Comment:
```suggestion
  return [date.getFullYear(), date.getMonth() + 1, date.getDate(), date.getHours(), date.getMinutes()];
```
Possibly gives the same result?

Reply:
I was actually thinking to implement it like that but didn't the `+ 1 ` part when defining months 😄 
Anyway, changed as suggested.